src/openforms/js/compiled-lang/nl.json

```diff
@@ -3886,6 +3886,12 @@
       "value": "Aantal dagen dat een sessie bewaard blijft. Laat leeg om de waarde van de algemene configuratie te gebruiken."
     }
   ],
+  "fGTdJz": [
+    {
+      "type": 0,
+      "value": "Het afleiden van adres via tekstveld-componenten is verouderd. Gebruik in de plaats de 'AddressNL'-component (in de 'Speciale velden' groep)."
+    }
+  ],
```

Comment:
The translation changes seem to be unrelated to this PR, possibly it was skipped for a previous PR?

Reply:
I'll double check - looks related to the formio-builder version :)

Reply:
ahh that makes sense, I was working on the addressNL validation as well, I'll see what happens if I reinstall

Reply:
You're right, I ran it again and now there are no changes, I've removed the commit